libminifi/include/agent/agent_docs.h

```diff
@@ -61,9 +63,12 @@ struct Components {
 };
 
 namespace detail {
-template<typename Container>
-auto toVector(const Container& container) {
-  return std::vector<typename Container::value_type>(container.begin(), container.end());
+inline auto toVector(std::span<const core::PropertyReference> properties) {
+  return std::vector<core::Property>(properties.begin(), properties.end());
+}
+
+inline auto toVector(std::span<const core::RelationshipDefinition> relationships) {
+  return std::vector<core::Relationship>(relationships.begin(), relationships.end());
```

Comment:
I'd use `span_to` here, instead of the local helpers. Before #1584, it's in utils/gsl.h, afterwards in utils/span.h

Reply:
This would mean changing
```c++
.class_properties_ = detail::toVector(Class::Properties)
```
to
```c++
.class_properties_ = utils::span_to<std::vector<core::Property>>(std::span<const core::PropertyReference>(Class::Properties))
```
in three places (plus one more for Relationships).  I'll do it if you think it's nicer, but I'm not sure it's nicer.